src/Stratis.Bitcoin.Features.PoA/FederationManager.cs

```diff
@@ -49,10 +46,15 @@ public void Initialize()
                 }
 
                 this.logger.LogInformation("Federation key pair was successfully loaded. Your public key is: {0}.", this.FederationMemberKey.PubKey);
-
-                // Enable mining because we are a federation member.
-                this.miner.InitializeMining();
             }
+
+            var builder = new StringBuilder();
```

Comment:
That will produce: 

```
[2018-10-09 10:55:04.4463 3] INFO: Stratis.Bitcoin.FullNode.Method
Federation contains...
[2018-10-09 10:55:04.4463 3] INFO: Stratis.Bitcoin.FullNode.Method
"Key1"
"Key2"
```

I want to avoid having that `[2018-10-09 10:55:04.4463 3] INFO: ` suffix on each line

Reply:
So:
```
this.logger.LogInformation($"Federation contains {this.network.FederationPublicKeys.Count} members. Their public keys are: {string.Join(Environment.NewLine, this.network.FederationPublicKeys)}");
```

Reply:
Good point. Fixed